socorro/unittest/external/elasticsearch/test_supersearch.py

```diff
@@ -24,11 +24,11 @@
 )
 
 # Remove debugging noise during development
```

Comment:
We know what this means, but it might be worth amplifying the comment to make it more clear to others.